packages/walletlink/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@web3-react/walletlink",
+  "publishConfig": {
+    "access": "public"
+  },
+  "version": "8.0.2-alpha.0",
+  "type": "module",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "prebuild": "rm -rf dist",
+    "build": "tsc",
+    "start": "tsc --watch"
+  },
+  "dependencies": {
+    "@web3-react/types": "^8.0.2-alpha.0",
+    "walletlink": "^2.1.11"
```

Comment:
can we make this a peer dep like in `@web3-react/walletconnect`? will also need to add this as a dependency to `example`. the motivation here is easier version upgrades for consumers

Reply:
sounds good, will change it now

Reply:
done